plugin.js

```diff
@@ -366,9 +386,9 @@
 		 */
 		this.addAll = function(placeholders)
 		{
-			suppressEvents = true;
-
+			suppressingEvents++;
```

Comment:
Whitespaces after operators please (same for the other ocurrences, including `--`).